packages/react/src/utils/useAnchorPositioning.ts

```diff
@@ -118,12 +122,29 @@ export function useAnchorPositioning(
   // presence.
   const arrowRef = React.useRef<Element | null>(null);
 
+  // Keep these reactive if they're not functions
+  const sideOffsetDep = typeof sideOffset !== 'function' ? sideOffset : 0;
+  const alignOffsetDep = typeof alignOffset !== 'function' ? alignOffset : 0;
+
```

Comment:
Positioning reactivity is lost when they specify a function. We can't tell if they're going to `React.useCallback` or not - usually the function gets inlined like:

```jsx
<Popover.Positioner sideOffset={(sizes) => sizes.anchor.height / 2}>
```

So it can't be specified as a dependency as it's always changing. This means if they update the math inside the function, the position won't update live on fast refresh (but it will whenever the anchoring updates at some point)